sympy/geometry/ellipse.py

```diff
@@ -1267,6 +1268,26 @@ def vradius(self):
         """
         return self.args[2]
 
+    @property
+    def centroid(self):
+        """Returns the centroid of ellipse.
+
+        Returns
+        =======
+
+        centroid : Point
+
+        Examples
+        ========
+        >>> from sympy import Point, Ellipse
+        >>> p1 = Point(0, 0)
+        >>> e1 = Ellipse(p1, 3, 1)
+        >>> e1.centroid
+        Point2D(0, 0)
+
+        """
+        return self.center
```

Comment:
I've commented on this before. I think we should have either centroid or center, not both. There is not reason to be redundant.

Reply:
@moorepants  yes,
Centre is used in the class itself almost everywhere. It is not good to remove the centre.

Reply:
Ok, so we should stick with center. There is no need to add a centroid property.